zbus/src/blocking/proxy/mod.rs

```diff
@@ -159,7 +159,7 @@ impl<'a> Proxy<'a> {
 
     /// Introspect the associated object, and return the XML description.
     ///
-    /// See the [xml](xml/index.html) module for parsing the result.
+    /// See the [xml](https://docs.rs/zbus_xml) module for parsing the result.
```

Comment:
oh and it's not a module anymore but a separate crate. :) Might as well correct that too. :)